### ISSUE TITLE
[AF3] Hide Locations selection step setting is not working 

### ISSUE THREAD

**Issue by ddrozdik** [ymcatwincities/openy_activity_finder#34]
**Test Environment:**
Environment: | https://sandboxes-d9.openy.org/
Drupal version | 9.0.10
Open Y version | 9.x-2.8.0
Open Y installation type | Custom
Open Y theme | Carnation

**Steps to reproduce:**
1. log in as admin 
2. Go to Content 
3. Find a page "Activity Finder"
4. Edit it
5. Enable checkbox "Hide Locations selection step"
![image](https://user-images.githubusercontent.com/6285570/107067498-69d04180-67f0-11eb-9507-e04b353851d1.png)
6. Save

**Expected result:**
When you walk through the activity finder steps you should not see the locations step. 

**Actual result:**
The locations step is visible and not hidden according to the setting from the paragraph. 
![image](https://user-images.githubusercontent.com/6285570/107067797-dd724e80-67f0-11eb-85ac-6f5948b6c7c9.png)

**Comment by ddrozdik** [comment]
cc @sarah-halby

**Comment by duozersk** [comment]
This one is not related to AF4 - it doesn't have this checkbox at all.
And when we were looking into porting this feature to AF4 - we found out that there is no implementation for this thing in AF3 - checkbox exists, but doesn't affect anything (there is nothing in the codebase reacting to it - looks like not finished backporting...).

**Comment by ddrozdik** [comment]
yes, however in code I see that there is a code that somehow was supposed to be used https://github.com/ymcatwincities/openy_activity_finder/blob/4.x/openy_af_vue_app/components/ActivityFinder.vue#L459, even in AFv4. That's why I marked it as related to AFv4 as well.

**Comment by duozersk** [comment]
The code reference above ^^ is for AF3.
